### DISCUSSION TITLE
aut

### DISCUSSION THREAD

**Discussion by Maybellynn** [community/community#60974] [Enterprise]
### Select Topic Area

Question

### Body

https://docs.github.com/en/enterprise-cloud@latest/get-started

**Comment by Akash1134** [comment]
Hi @Maybellynn  , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

Do you have question or feedback regarding the "Get started with GitHub documentation" for Enterprise Cloud?

Thanks!
